partition/roles/monitoring/ipmi-exporter/README.md

```diff
@@ -0,0 +1,12 @@
+# ipmi-exporter
+
+Deploys the ipmi-exporter in a systemd-managed Docker container.
+
+## Variables
+
```

Comment:
We need to raise a follow-up PR. This role depends on `metal_api_bmc_superuser_pwd`, introducing variable cross-dependency. We need to introduce a new variable `monitoring_ipmi_exporter_bmc_superuser_pwd` and can try to default this to `metal_api_bmc_superuser_pwd`. For this PR it's fine.

Reply:
https://github.com/metal-stack/metal-roles/issues/364